src/actions/heap/heap.ts

```diff
@@ -155,12 +157,17 @@ export class HeapAction extends Hub.Action {
       errors.push(err)
     }
 
-    await this.trackLookerAction(
-      request.params.heap_env_id!,
-      requestCount,
-      heapField,
-      errors.length === 0 ? "success" : "failure",
-    )
+    try {
+      await this.trackLookerAction(
+        request.params.heap_env_id!,
+        requestCount,
+        heapField,
+        errors.length === 0 ? "success" : "failure",
+      )
+    } catch (err) {
+      // swallow internal track call error
```

Comment:
What do you think about logging this error. It seems like this project is using winston to log: https://github.com/heap/actions/blob/601186b3fd9c30336e0c1b9ccd962f340f9002db/src/actions/debug/debug.ts#L26

Reply:
Sure we can add some more logging